scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta

```diff
@@ -12,8 +12,9 @@ public class <%= it.className %> implements ModInitializer {
 	// This logger is used to write text to the console and the log file.
 	// It is considered best practice to use your mod id as the logger's name.
 	// That way, it's clear which mod wrote info, warnings, and errors.
-<% if (it.slf4j) { %>    public static final Logger LOGGER = LoggerFactory.getLogger("<%= it.modid %>");
-<% } else { %>    public static final Logger LOGGER = LogManager.getLogger("<%= it.modid %>");<% } %>
+        public static final String MOD_ID = "<%= it.modid %>";
+<% if (it.slf4j) { %> public static final Logger LOGGER = LoggerFactory.getLogger("<%= MOD_ID %>");
+<% } else { %> public static final Logger LOGGER = LogManager.getLogger("<%= MOD_ID %>");<% } %>
```

Comment:
Indentation needs to be fixed

Reply:
Yes, please double check this, its quite finicky to get this right.

Reply:
Can you give me an example to how it should look like ?